scripts/allRoutesTest.ts

```diff
@@ -86,9 +86,12 @@ async function main() {
     const wbtc = await ethers.getContractAt("IERC20Metadata", "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599");
     const fxs = await ethers.getContractAt("IERC20Metadata", "0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0");
     const fraxBP = await ethers.getContractAt("IERC20Metadata", "0xe57180685e3348589e9521aa53af0bcd497e884d");
+    const cvxCrvFrax = await ethers.getContractAt("IERC20Metadata", "0x527331f3f550f6f85acfecab9cc0889180c6f1d5");

```

Comment:
cvxCrvFrax and cvxCrvFraxLP are same token variables here. Please remove either of those